programs/bpf_loader/src/helpers.rs

```diff
@@ -404,10 +403,10 @@ mod tests {
         .unwrap();
     }
 
-    // Serialize this test: solana_logger conflicts when running tests concurrently,
+    // ignpre this test: solana_logger conflicts when running tests concurrently,
```

Comment:
Ignore?

Reply:
Yeah, tried to serialize these tests but ran into the issue again locally when running all the tests.  I have another PR where I'm going to ignore these tests until I can figure out how to address the concurrency conflict. 
https://github.com/solana-labs/solana/pull/9719

Reply:
All good, just pointing out the typo